data-structures/hashmap.md

```diff
@@ -332,21 +320,10 @@ type Bucket [ BucketEntry ]
 
 type BucketEntry struct {
   key Bytes
-  value Value
+  # There is currently no limitation on the types available for storage as long
```

Comment:
Tiniest nit: I think this would read a little better as prose after this code block than as a comment on the value field.

Reply:
done